linux_os/guide/system/accounts/accounts-restrictions/root_logins/no_direct_root_logins/oval/shared.xml

```diff
@@ -8,19 +8,25 @@
     </criteria>
   </definition>
 
-  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="/etc/securetty file exists" id="test_etc_securetty_exists" version="1">
+  <ind:textfilecontent54_test id="test_etc_securetty_exists" version="1"
+      check="all" check_existence="all_exist" comment="/etc/securetty file exists">
     <ind:object object_ref="object_etc_securetty_exists" />
   </ind:textfilecontent54_test>
-  <ind:textfilecontent54_object comment="/etc/securetty file exists" id="object_etc_securetty_exists" version="1">
+
+  <ind:textfilecontent54_object id="object_etc_securetty_exists" version="1"
+      comment="/etc/securetty file exists">
```

Comment:
Small nitpick: We usually avoid checking whitespace changes in, because it just adds noise during the review, the reviewer needs to examine what exactly changes there. Don't remove them now but try avoiding it in future PRs.

Reply:
I'm aware @jan-cerny . But as it was a very small change I chose to keep them with the other changes.